org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java

```diff
@@ -1805,8 +1817,24 @@ public double getAttribute (String conceptValue, Timestamp breakDate) {
 			return attribute.getQty().doubleValue();
 
 		// if column type is Amount return amount
-		if (concept.getColumnType().equals(MHRConcept.COLUMNTYPE_Amount))
-			return attribute.getAmount().doubleValue();
+		if (concept.getColumnType().equals(MHRConcept.COLUMNTYPE_Amount)) {
+			BigDecimal rate = Env.ONE;
+			BigDecimal amount = attribute.getAmount();
+			if(attribute.isConvertedAmount() && attribute.getC_Currency_ID() != getC_Currency_ID()) {
+				int currencyPrecision = MCurrency.getStdPrecision(getCtx(), Env.getContextAsInt(p_ctx, "#C_Currency_ID"));
+				Optional<Integer> conceptStandardPrecisionOptional = Optional.ofNullable((Integer)concept.get_Value("StdPrecision"));
```

Comment:
@yamelsenih please change the concept.get_Value("StdPrecision") by the generate model

Reply:
I will to change. I also will change it for old implementation of round of concept